packages/shared/lib/core/profile-manager/api/events/spentOutput.ts

```diff
@@ -0,0 +1,6 @@
+import { OutputData } from '@iota/wallet'
+import { syncBalance } from '@core/account/actions/syncBalance'
+
+export async function handleSpentOutput(accountId: string, payload: { output: OutputData }): Promise<void> {
+    await syncBalance(accountId)
+}
```

Comment:
We need to update the activity as well to determine if an async transaction has been claimed by the recipient

Reply:
https://github.com/iotaledger/firefly/pull/3478/commits/285c93e8c1de1e88b52cff7327d060da7eae838d